tests/copy-paste.js

```diff
@@ -989,6 +989,22 @@ define([
             assert(!util.markdownVisible());
         });
 
+        it("should paste invalid xpaths correctly", function() {
+            util.loadXML("");
+            paste([
+                ["id", "type", "calculateAttr"],
+                ["/invalid", "DataBindOnly", "(42"],
+                ["/invalid2", "DataBindOnly", "#invalid/xpath (42"],
+            ]);
+            util.clickQuestion('invalid');
```

Comment:
seems like it might be worth checking the value of `util.getMug("invalid").p.calculateAttr` and also the Calculate Condition input field to verify that they do or do not have the `#invalid/xpath ` prefix (I'm not sure which is correct in each case).

Also might be worth adding a test with an escaped hashtag in the (invalid) expression to verify that that is preserved and properly converted to a bubble in the text input on paste.